src/components/Categories/Categories.jsx

```diff
@@ -1,36 +1,111 @@
 import React from 'react';
 import './Categories.css'
 import Button from 'react-bootstrap/Button';
-
-import {Link, Outlet}  from "react-router-dom";
+import { useEffect, useState } from 'react';
+import Axios from 'axios';
+import { CartState } from '../../Context/Context';
+import Card from 'react-bootstrap/Card';
 export default function Categories() {
+  const {state:{cart},
+  dispatch
+ }= CartState();
+  const [categories,setCategories] = useState([]);
+  const [items,setItems] = useState([]);
+  const [filteredItems, setFilteredItems] = useState(items);
+  const [success,setSuccess] = useState(false);
+  async function getCategories(){
+    let {data} = await Axios.get(`http://localhost:9999/api/categories`);
```

Comment:
move any url to .env file then import it from there (we do that for deployment or change purpose)